benchmark/lora/lora_bench.py

```diff
@@ -1,17 +1,16 @@
-"""
```

Comment:
Could you elaborate more about what the word "effective" means?

If the module do not have a docstring, the `__doc__` attribute should be `None` instead of a copyright string.

Reply:
Hi @XuehaiPan I just realized I didn't look carefully before. Thank you for your reply. Changing """ to # is reasonable. Could you send another PR? Thanks!